### ISSUE TITLE
Phantom boards

### ISSUE THREAD

**Issue by emanuele45** [elkarte/Elkarte#1296]
The idea is boards that exist, but are not listed in board index or the child-boards or jump-to blocks, etc.

Reason?
Very limited usefulness, that's why I opened an issue instead of coding them. :angel: 
I just had to delete a board that once contained something. The something has been moved to another board, but now the old links (search engines, bookmarks, etc.) are all invalid. With a phantom board I could have changed the old board to a redirect invible and obtain a "clean" redirect.

**Comment by emanuele45** [comment]
TBH I'm not entirely sure what I was talking about...

**Comment by Spuds** [comment]
LOL

**Comment by wintstar** [comment]
I like this idea. Each user could then set up his own "board" with his personal topic.

+1

**Comment by emanuele45** [comment]
@wintstar that was not about giving users the possibility to create boards, but I guess more about having multiple types of boards (that actually could be used for what you were suggesting as well), though I wrote it in a way that doesn't really make much sense.
The only way it somehow make sense is if I was referring to links to the board itself (```board=1.0```), that disappear when a board is deleted, while could work as a redirect with the setting I described above (i.e. board 1 remains in an hidden form and acts as a redirect to another board, for example 2, so that links pointing to 1 will not receive a 404, but a 301).

**Comment by wintstar** [comment]
Yes, a brilliant idea. I think that you could make of this idea a little bit more.
